app/models/group.js

```diff
@@ -33,16 +31,21 @@ export default Model.extend(AvatarModelMixin, {
     }
 
     return null;
-  }),
-  alternativeGroupLogoText: computed('name', function() {
-    return `Het logo van ${this.name}`;
-  }),
+  }
+
+  get avatarUrlOrDefault() {
+    return this.avatarUrl || '/images/fallback/group_avatar_default.png';
```

Comment:
Why did you not make this a constant like the other default images?

Reply:
because this image is only used for groups, the other are (as far as I remember) used at multiple locations